x-pack/plugins/fleet/server/constants/fleet_es_assets.ts

```diff
@@ -36,10 +40,14 @@ export const FLEET_GLOBAL_COMPONENT_TEMPLATE_CONTENT = {
   },
 };
 
-export const FLEET_FINAL_PIPELINE_VERSION = 1;
+export const FLEET_FINAL_PIPELINE_VERSION = 2;
```

Comment:
@nchaulet - I wasn't 100% sure whether we should increment this pipeline version since we're adding `_meta` below.

Reply:
I think incrementing the version will work it will trigger to update the pipeline to the latest version on fleet setup.